### PR TITLE
 JavaScript Course Project Recursion: Add and Reformat Test Examples

### PR DESCRIPTION
**Pull request by karimdevelops**
## Because

I have added three more test examples including an **empty** Array, **single** element and **already sorted** one. Also, the following text kinda works but I think it will be more readable and easier to copy if we write it in bullet points.

> An input of [3, 2, 1, 13, 8, 5, 0, 1] should return [0, 1, 1, 2, 3, 5, 8, 13], and an input of [105, 79, 100, 110] should return [79, 100, 105, 110].

**To**
Test Examples:

- `mergeSort([])` → `[]`
- `mergeSort([73])` → `[73]`
- `mergeSort([1, 2, 3, 4, 5])` → `[1, 2, 3, 4, 5]`
- `mergeSort([3, 2, 1, 13, 8, 5, 0, 1])` → `[0, 1, 1, 2, 3, 5, 8, 13]`
- `mergeSort([105, 79, 100, 110]` → `[79, 100, 105, 110]`

These tests are good, and I failed the first two myself.

## This PR

- Add new test examples
- Reformat test examples

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by karimdevelops** [comment]
I am glad to contribute to TOP, all changes are done now. This time especially this merge sort one was completely my own and a bigger contribution compared to my very first one which was about `Lorem Ipsum` shortcut in vs code haha.

File: ruby/computer_science/project_recursion.md [hunk 1]
```diff
@@ -37,7 +37,15 @@ Hopefully you were able to solve the problem with recursion! If you need some he
 
 Once you have a firm grasp on solving Fibonacci with recursion, create a new file and work on a merge sort:
 
-1. Build a method `#merge_sort` that takes in an array and returns a sorted array, using a recursive merge sort methodology. An input of `[3, 2, 1, 13, 8, 5, 0, 1]` should return `[0, 1, 1, 2, 3, 5, 8, 13]`, and an input of `[105, 79, 100, 110]` should return `[79, 100, 105, 110]`.
+1. Build a function `mergeSort` that takes in an array and returns a sorted array, using a recursive merge sort methodology.
```
Comment (JoshDevHub) [inline ruby/computer_science/project_recursion.md]:
```suggestion
1. Build a function `#merge_sort` that takes in an array and returns a sorted array, using a recursive merge sort methodology.
```
Just reverting the change here to stick to Ruby's naming conventions.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Just one small fix that I forgot to mention related to Ruby method references, but after that this will be good to approve and merge. Thank you for the work you've done on these computer science lessons 🙌

File: ruby/computer_science/project_recursion.md (hunk 1 above)
Comment (karimdevelops) [reply ruby/computer_science/project_recursion.md]: Ehhh, I missed this one

**Review by JoshDevHub — APPROVED** [review]
